jetstream/resource_kv_bucket.go

```diff
@@ -249,13 +270,15 @@ func resourceKVBucketUpdate(d *schema.ResourceData, m any) error {
 	maxV := d.Get("max_value_size").(int)
 	maxB := d.Get("max_bucket_size").(int)
 	description := d.Get("description").(string)
+	markerTTL := d.Get("limit_marker_ttl").(int)
 
 	cfg := str.Configuration()
 	cfg.MaxAge = time.Duration(ttl) * time.Second
 	cfg.MaxMsgSize = int32(maxV)
 	cfg.MaxBytes = int64(maxB)
 	cfg.MaxMsgsPer = int64(history)
 	cfg.Description = description
+	cfg.SubjectDeleteMarkerTTL = time.Duration(markerTTL) * time.Second
```

Comment:
here we should use the kv update function instead of using the str loaded from jsm.